test/fixtures/all-fields-populated/package.json

```diff
@@ -0,0 +1,96 @@
+{
```

Comment:
the existing `test/fixtures/package.json` seems out of place now, perhaps it can be moved into its own subfolder also,